protocol/src/main/protobuf/Transactor.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+package protocol;
+option java_package = "io.mediachain.protocol.transactor";
+
+message MultihashReference {
+    string reference = 1;
+}
+
+message InsertRequest {
+    bytes canonicalCbor = 1;
+}
+
+message UpdateRequest {
+    bytes chainCellCbor = 1;
+}
+
+service TransactorService {
+    rpc FetchObjectChainHead(MultihashReference) returns
```

Comment:
my issue with "updateChain/lookupChain" is that people seem to get the canonical chains and the blockchain confused (see my comments here https://github.com/mediachain/mediachain/pull/109#discussion_r65455130)